default_env_value/default_env_value.go

```diff
@@ -0,0 +1,11 @@
+package default_env_value
+
+import "os"
+
+func SetDefaultEnvValue(key, defaultVal string) string {
+	value, exists := os.LookupEnv(key)
+	if !exists {
+		value = defaultVal
+	}
+	return value
+}
```

Comment:
When I saw **Set**DefaultEnvValue in the client code I thought this was actually setting an environment variable.

Maybe rename to something like `GetWithDefault()` or something like that.

Also, may tweak the if could read a lil bit better:

```golang
func GetWithDefault(key, defaultVal string) string {
	value, exists := os.LookupEnv(key)
	if exists {
		return value
	}
	return defaultVal
}
```